roles/python-3/tasks/install.yml

```diff
@@ -68,3 +67,9 @@
       - { target: "/usr/local/bin/{{ python_3_executable }}", link: "/usr/bin/python3" }
       - { target: "/usr/local/bin/{{ pip_3_executable }}", link: "/usr/bin/pip3" }
     when: ansible_distribution == "CentOS"
+
+  - name: upgrade pip3 to specified version
+    pip:
+      executable: pip3
```

Comment:
If there any role depends on `pip2`?  How would we use `pip2` if we need to?

Reply:
If we want to use `pip2`, we should replace `pip3` with `pip2` in the `executable` field for this module. From some testing I did, using the `pip` ansible module without specifying the executable should still use `pip2`, even after installing `pip3` and overwriting the `pip` binary. So this shouldn't be a breaking change for our older roles that don't specify the executable name. Since we use python 3 and pip 3 along with python 2 and pip 2 now, we should probably specify the executable name when using the pip module in future roles/changes.